src/components/TableBody.js

```diff
@@ -210,8 +210,21 @@ class TableBody extends React.Component {
     this.props.options.onRowClick && this.props.options.onRowClick(row, data, event);
   };
 
+  processRow = (row, columnOrder) => {
+    var ret = [];
+
+    for (var ii = 0; ii < row.length; ii++) {
+      ret.push({
+        value: row[ columnOrder[ii] ],
+        index: columnOrder[ii]
+      });
+    }
+
+    return ret;
```

Comment:
It would be better to reduce the use of (var, let).

Especially in the case of **'var'**, you should avoid using it.

In the case below, it seems to be unnecessary use (var).
The behavior is the same even if you change to **const**.
Because the reference point of the variable has not changed.

```js
const ret = [];
ret.push(1); // [1]
console.log(ret) // [1]

var ret = [];
ret.push(1); // [1]
console.log(ret) // [1]
```

I think it will disappear if webpack is applied, 
but I think it would be better to use it properly from the beginning. 

thanks

Reply:
Agreed, I'll go back and replace the vars, old habits die hard :P